### PR TITLE
Feature/initial api test framework

### PR DESCRIPTION
**Pull request by stewartie4**
Rough initial framework to create wallet + allocation, ready for additional API tests

### CODE REVIEW

File: internal/api/util/Config.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+	"io/ioutil"
+
+	"gopkg.in/yaml.v3"
+)
```
Comment (stewartie4) [inline internal/api/util/Config.go:7]: this is clearly used on line 18

File: internal/api/util/Config.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	NetworkEntrypoint string `yaml:"network_entrypoint"`
+}
+
+func (config *Config) Init(path string) {
+	yamlFile, err := ioutil.ReadFile(path)
+	if err != nil {
+		panic("Failed to read config file!")
+	}
+	err = yaml.Unmarshal(yamlFile, config)
+	if err != nil {
```
Comment (stewartie4) [inline internal/api/util/Config.go:19]: see import

Comment (stewartie4) [reply internal/api/util/Config.go:19]: the rest of these are the same... i'm obviously doing something wrong

File: internal/api/model/Model.go [hunk 3]
```diff
@@ -1,7 +1,11 @@
-package apimodel
+package model
```
Comment (Kishan-Dhakan) [inline internal/api/model/Model.go:1]: Why rename this file to titlecase?😅 Conventionally go files are snake case or all lower case names, although this is not documented strictly anywhere. Even so, to be consistent in this repo we could stick to one style

**Review by Kishan-Dhakan — APPROVED** [review]
LGTM

File: internal/api/model/Model.go (hunk 3 above)
Comment (stewartie4) [reply internal/api/model/Model.go:1]: I'm used to java i guess lol i'll revert it

**Pull request merged**